src/client/pages/Offer/index.tsx

```diff
@@ -210,6 +224,10 @@ export const OfferPage = ({
           <AppPromotionSection />
           <FaqSection />
           <Checkout
+            quoteCartId={quoteCartId}
```

Comment:
[Suggestion]
`quoteCartId` can be retrieved inside `<Checkout />` with `const { id: quoteCartId } = useParams<{ id: string }>()`.

Reply:
Sure, but we just wanna keep it simple :) it will rerender properly either way